data/json/items/tool_armor.json

```diff
@@ -426,6 +426,40 @@
         "material_thickness" : 1
     },
     {
+        "id": "binoculars",
+        "type": "TOOL_ARMOR",
+        "category" : "clothing",
+        "symbol": ";",
+        "color": "grey",
```

Comment:
It should be ```gray``` (if you want to see this color) or ```dark_gray``` (if you want to keep color used for binoculars previously).